src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,296 @@
+        } catch (NumberFormatException e) {
+            sb.append(ui.printErrorTaskNumber());
+        } catch (java.time.format.DateTimeParseException e) {
+            sb.append(ui.printErrorDatetimeFormat());
+        } finally {
+            ui.printLineBreak();
+        }
+        if (sb.toString().equals("")) {
+            return "Sorry! I don't know how to respond to that";
+        }
+        return sb.toString();
+    }
+
+//    @Override
+//    public void start(Stage stage) {
+//        //Step 1. Setting up required components
+////
+////        //The container for the content of the chat to scroll.
+////        scrollPane = new ScrollPane();
+////        dialogContainer = new VBox();
+////        scrollPane.setContent(dialogContainer);
+////
+////        userInput = new TextField();
+////        sendButton = new Button("SendMEEE");
+////
+////        AnchorPane mainLayout = new AnchorPane();
+////        mainLayout.getChildren().addAll(scrollPane, userInput, sendButton);
+////
+////        stage.setTitle("Duke");
+////        stage.setResizable(false);
+////        stage.setMinHeight(600.0);
+////        stage.setMinWidth(400.0);
+////
+////        mainLayout.setPrefSize(400.0, 600.0);
+////
+////        scrollPane.setPrefSize(385, 535);
+////        scrollPane.setHbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
+////        scrollPane.setVbarPolicy(ScrollPane.ScrollBarPolicy.ALWAYS);
+////
+////        scrollPane.setVvalue(1.0);
+////        scrollPane.setFitToWidth(true);
+////
+////        // You will need to import `javafx.scene.layout.Region` for this.
+////        dialogContainer.setPrefHeight(Region.USE_COMPUTED_SIZE);
+////
+////        userInput.setPrefWidth(325.0);
+////
+////        sendButton.setPrefWidth(55.0);
+////
+////        AnchorPane.setTopAnchor(scrollPane, 1.0);
+////
+////        AnchorPane.setBottomAnchor(sendButton, 1.0);
+////        AnchorPane.setRightAnchor(sendButton, 1.0);
+////
+////        AnchorPane.setLeftAnchor(userInput, 1.0);
+////        AnchorPane.setBottomAnchor(userInput, 1.0);
+////
+////        sendButton.setOnMouseClicked((event) -> {
+////            dialogContainer.getChildren().add(getDialogLabel(userInput.getText()));
+////            userInput.clear();
+////        });
+////
+////        userInput.setOnAction((event) -> {
+////            dialogContainer.getChildren().add(getDialogLabel(userInput.getText()));
+////            userInput.clear();
+////        });
+//
+//        //Part 3. Add functionality to handle user input.
+////        sendButton.setOnMouseClicked((event) -> {
+////            handleUserInput();
+////        });
+////
+////        userInput.setOnAction((event) -> {
+////            handleUserInput();
+////        });
+////
+////        dialogContainer.heightProperty().addListener((observable) -> scrollPane.setVvalue(1.0));
+////
+////        scene = new Scene(mainLayout);
+////        stage.setScene(scene);
+////        stage.show();
+//    }
```

Comment:
Consider removing this function as soon as you're certain it won't be needed anymore, as well as other occurrences in code.